src/components/ContentHighlights/CatalogVisibility/ContentHighlightCatalogVisibilityRadioInput.jsx

```diff
@@ -32,21 +34,21 @@ const ContentHighlightCatalogVisibilityRadioInput = () => {
    * Sets enterpriseCuration.canOnlyViewHighlightSets to false if there are no highlight sets
    * when the user enters content highlights dashboard.
    */
+  const setDefault = useCallback(async () => {
```

Comment:
[curious] Can you expand on the rationale for this change? Why can't we define `setDefault` within the `useEffect` here?